src/app/components/todos.component/todos.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnChanges, OnDestroy, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import 'rxjs/add/operator/map';
+
+import { TodosActions } from '../../actions/todos.actions';
+import { TodosState } from '../../reducers/totos.reducer';
+import { Todo } from '../../models/todo';
+import { Observable } from 'rxjs/Observable';
+import { Subscription } from 'rxjs/Subscription';
+
+@Component({
+  selector: 'app-todos',
+  templateUrl: './todos.component.html'
+})
+export class TodosComponent implements OnInit, OnDestroy {
+  textTodo: string;
+  todos: Todo[];
+  state: Observable<TodosState>;
+  stateSubscription: Subscription;
+
+  constructor(private store: Store<TodosState>, private actions: TodosActions) {
+    this.textTodo = '';
+    this.todos = [];
+    this.state = this.store.select('todos');
+    this.store.dispatch(this.actions.getTodos());
+  }
+
+  ngOnInit() {
+    this.stateSubscription = this.state.subscribe(state => {
```

Comment:
you don't need to subscribe in order to show todos -> have a look at `async` pipe